pkgs/racket-doc/scribblings/reference/eval-model.scrbl

```diff
@@ -556,9 +556,11 @@ module-specific prefix).
 @;------------------------------------------------------------------------
 @subsection[#:tag "module-phase"]{Phases}
 
+@guidealso["phases"]
+
 The purpose of @deftech{phases} is to
 address the necessary separation of names defined at execution time versus
-names defined at expansion time. See Section @secref["phases"] in the Guide.
+names defined at expansion time.
```

Comment:
The current Reference style is that it never depends on the Guide to define anything. Instead, it always cross-references the Guide in a margin notes. That's not obviously the right style choice, but we reconsider it, we'd probably want to reconsider more globally. Also, I don't think `@secref["phases"]` was going to the right place, since a `#:doc` argument is needed to link to another document.